Servers/UI/OJS.Servers.Ui.Models/Submissions/Details/SubmissionTypeForSubmissionDetailsResponseModel.cs

```diff
@@ -7,4 +7,6 @@ public class
     SubmissionTypeForSubmissionDetailsResponseModel : IMapFrom<SubmissionTypeForSubmissionDetailsServiceModel>
 {
     public bool AllowBinaryFilesUpload { get; set; }
+
+    public string Name { get; set; } = null!;
```

Comment:
Better use `= string.Empty;` instead of null, this way Name will never be null, as it should be by definition, as it is declared as non nullable string. Or define it as `string?` if nulls are expected and allowed here.

Reply:
Fixing in the service (model it is mapped from) and the response model